main.py

```diff
@@ -114,28 +117,28 @@ def key(message):
     key_name = cmd_get_rest(message.text)
     run_command_and_notify(message, ['xdotool', 'key', key_name], expect_quick=True)
 
-@bot.message_handler(func=lambda message: cmd_get_action(message.text) in ['exec', 'rawexec'])
+@bot.message_handler(func=lambda message: cmd_get_action(message.text) in ['run', 'rawrun', 'exec', 'rawexec'])
 @admins_only_handler
-def exec_raw_exec(message):
-    action = cmd_get_action(message.text)
-    to_exec = cmd_get_rest(message.text)
-    if action == 'exec':
+def run_raw_run(message):
+    action = cmd_get_action(message.text).replace('exec', 'run')
+    to_run = cmd_get_rest(message.text)
+    if action == 'run':
         try:
-            to_exec = shlex.split(to_exec)
+            to_run = shlex.split(to_run)
         except ValueError as e:
             bot.reply_to(message, f"Failed:\n{e}")
```

Comment:
I suppose, this can also be fixed as a miscellaneous issue: the error message should be a little more verbatim than just "failed", even though some information about the exception is provided